common/app/common/templates/HandlebarsTemplateService.java

```diff
@@ -42,17 +59,38 @@ private Template compileTemplate(final String templateName) {
         }
     }
 
-    private Context buildContext(final PageData pageData) {
+    private Context buildContext(final PageData pageData, final String templateName) {
         // TODO Use resolver with cache on production
-        return Context.newBuilder(pageData)
-                .resolver(NonCachedJavaBeanValueResolver.INSTANCE)
-                .build();
+        Context.Builder builder = Context.newBuilder(pageData)
+                .resolver(NonCachedJavaBeanValueResolver.INSTANCE, MapValueResolver.INSTANCE);
```

Comment:
No, you can see the TODO right above this commit :P
Actually, I'll create the task so I don't forget :)